### PR TITLE
Bra 25 homework

### PR DESCRIPTION
**Pull request by dev-kucha**
Lesson 25

### CODE REVIEW

File: js/lesson-25/part10.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+`use strict`;
+
+function getTimeSeconds(h, m, s) {
```
Comment (roskyba) [inline js/lesson-25/part10.js:3]: Wee need to check if user enter valid numbers for date  `h < 25 m < 59 s <59`

File: js/lesson-25/part10.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+function getTimeString(s) {
+    let now = new Date(0, 0, 0, 0, 0, s);
+    return now.toLocaleTimeString();
+}
+
+function getDifferenceOfTimes(h1, m1, s1, h2, m2, s2) {
+    let time1 = new Date(0, 0, 0, h1, m1, s1);
+    let time2 = new Date(0, 0, 0, h2, m2, s2);
+    let difference = Math.abs(getTimeSeconds(h1, m1, s1) - getTimeSeconds(h2, m2, s2));
```
Comment (roskyba) [inline js/lesson-25/part10.js]: Good solution with `Math.abs()`

File: js/lesson-25/part4.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+`use strict`;
+
+function calcAreaOfRectangle(a, b) {
+    return Number(a) * Number(b || a);
```
Comment (roskyba) [inline js/lesson-25/part4.js:4]: Awesome solution!

File: js/lesson-25/part5.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    if (x <= 0) {
+        return false;
+    }
+    let sum = 0;
+    for (let i = 1; i < x; i++) {
+        if (x % i == 0) {
+            sum += i;
+        }
+    }
+    return (x == sum);
```
Comment (roskyba) [inline js/lesson-25/part5.js]: You can ommit () in x == sum

File: js/lesson-25/part6.js [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    let sum = 0;
+    for (let i = 1; i < x; i++) {
+        if (x % i == 0) {
+            sum += i;
+        }
+    }
+    return (x == sum);
+}
+
+function getPerfectNubdersOfRange(a, b) {
```
Comment (roskyba) [inline js/lesson-25/part6.js]: Nubders ==> Numbers

File: js/lesson-25/part7.js [hunk 6]
```diff
@@ -0,0 +1,12 @@
+`use strict`;
+
+function showTime(h, m = '00', s = '00') {
```
Comment (roskyba) [inline js/lesson-25/part7.js:3]: Good point with default numbers!

File: js/lesson-25/part10.js (hunk 1 above)
Comment (dev-kucha) [reply js/lesson-25/part10.js:3]: Fixed

File: js/lesson-25/part10.js (hunk 2 above)
Comment (dev-kucha) [reply js/lesson-25/part10.js]: :-)

File: js/lesson-25/part4.js (hunk 3 above)
Comment (dev-kucha) [reply js/lesson-25/part4.js:4]: :-)

File: js/lesson-25/part5.js (hunk 4 above)
Comment (dev-kucha) [reply js/lesson-25/part5.js]: Fixed

File: js/lesson-25/part6.js (hunk 5 above)
Comment (dev-kucha) [reply js/lesson-25/part6.js]: Fixed

File: js/lesson-25/part7.js (hunk 6 above)
Comment (dev-kucha) [reply js/lesson-25/part7.js:3]: :-)

**Review by roskyba — APPROVED** [review]

**Pull request merged**